src/EthPool.sol

```diff
@@ -92,6 +92,20 @@ contract EthPool is Pool {
         _giveFromSubSenderInternal(msg.sender, subSenderId, receiver, uint128(msg.value));
     }
 
+    /// @notice Sets a new list of drips receivers of the sender of the message.
+    /// @param currReceivers The list of the user's drips receivers which is currently in use.
+    /// If this function is called for the first time for the user, should be an empty array.
+    /// @param newReceivers The new list of the user's drips receivers.
+    /// Must be sorted by the drips receivers' addresses, deduplicated and without 0 weights.
+    /// Each drips receiver will be getting `weight / TOTAL_DRIPS_WEIGHTS`
+    /// share of the funds collected by the user.
+    function setDripsReceivers(
```

Comment:
Why is this method on the ETHPool and ERC20 level? Both are doing the same could we not move it to the pool?

Reply:
We could, but then `Pool` would be dealing with `msg.sender`. We want to keep that separation and let the non-abstract pools decide which user is being updated.